spec/stack_master/commands/apply_spec.rb

```diff
@@ -135,6 +136,21 @@ def apply
         expect(StackMaster::ChangeSet).to_not have_received(:execute).with(change_set.id)
       end
     end
+
+    context 'yes_param option is set' do
+      let(:yes_param) { 'YesParam' }
+      let(:options) { double(yes_param: yes_param).as_null_object }
+
+      before do
+        allow(StackMaster).to receive(:non_interactive_answer).and_return('n')
+        allow(differ).to receive(:single_param_update?).with(yes_param).and_return(true)
+      end
+
+      it "skips asking for confirmation on single param updates" do
```

Comment:
That is tested in the unit test for `Differ#single_param_update?` [below](https://github.com/envato/stack_master/pull/252/files#diff-09b5c01e5802f5b0696133fee8aa723aR88), along with various other input combinations.

Here I'm just stubbing `Differ#single_param_update?`, so that I don't have to retest all those combinations here.

It is stubbed to `false` by default in all the other tests of stack update above this one, so the scenarios where it returns false are covered there. (it can be multiple param updates, but also other combinations like no `--yes-param` set, removal of the yes param, etc.)